experimental/PropertyDDS/packages/property-query/CHANGELOG.md

```diff
@@ -23,6 +23,8 @@
     **Node10 moduleResolution is not supported; it does not support Fluid Framework's API structuring pattern that is used
     to distinguish stable APIs from those that are in development.**
 
+-   `joi` dependency was updated from 14.3.1 to 17.3.1 to address a critical vulnerability exploit [CVE-2020-36604](https://github.com/advisories/GHSA-c429-5p7v-vgjp). This required updating the use of `joi` schema validation function within `property-query` to the new major version syntax.
```

Comment:
This needs to go into a .changeset file. The release process (for the client release group, specifically) eventually generates release notes and updates individual package changelogs based on the changesets. [This wiki page](https://github.com/microsoft/FluidFramework/wiki/Changesets) should guide you through creating one if necessary.

Reply:
Updated.